framework/logging.cpp

```diff
@@ -2344,7 +2351,7 @@ inline int YamlLogger::print_one_thread_messages(int fd, mmap_region r, int leve
                 // Only print if the result line didn't already include this
                 std::string skip_message;
                 format_skip_message(skip_message, message);
-                format_and_print_message(fd, 4, std::string_view{&skip_message[0], skip_message.size()}, true);
+                format_and_print_skip_message(fd, skip_message);
```

Comment:
I think when `format_and_print_skip_message` function is called here, it prints the wrong message when there is a skip in the run function.

Current message format would be:
```
  threads:                                                                                                                                                                                              
  - thread: 0                                                                                                                                                                                           
    id: { logical:  0, package: 0, core:  0, thread: 0, family: 6, model: 0x55, stepping: 6, microcode: 0x400320a, ppin: null }                                                                         
    messages:                                                                                                                                                                                           
    - { level: skip, text: 'CPU' }       
```
With this change, the format would be:
```
 threads:                                 
  - thread: 0                              
    id: { logical:  0, package: 0, core:  0, thread: 0, family: 6, model: 0x55, stepping: 6, microcode: 0x400320a, ppin: null }
    messages:                              
  skip-reason: 'CPU' 
```

Reply:
So it does. That means this wasn't properly tested in Bats.

Reply:
I take that back, it did fail here. I don't know why I didn't see it.